java/app/src/main/java/com/google/android/gms/samples/wallet/CheckoutActivity.java

```diff
@@ -192,53 +222,44 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
    * requested information, such as billing and shipping address.
    *
    * @param paymentData A response object returned by Google after a payer approves payment.
-   * @see <a
-   *     href="https://developers.google.com/pay/api/android/reference/object#PaymentData">Payment
-   *     Data</a>
+   * @see <a href="https://developers.google.com/pay/api/android/reference/
+   *      object#PaymentData">PaymentData</a>
    */
   private void handlePaymentSuccess(PaymentData paymentData) {
-    String paymentInformation = paymentData.toJson();
 
     // Token will be null if PaymentDataRequest was not constructed using fromJson(String).
-    if (paymentInformation == null) {
-      return;
-    }
-    JSONObject paymentMethodData;
+    final String paymentInfo = paymentData.toJson();
+    if (paymentInfo == null) return;
```

Comment:
Not sure what the convention is, but further down the file (line 292) I see a 1-line if statement with a return that includes brackets, and this one doesn't, so we should use one style or the other. My preference would be to include brackets, eg: 

```
if (paymentInfo == null) {
    return;
}
```

Reply:
Adding